### DISCUSSION TITLE
[ZK Stack] - <block on L1 is stuck in the "validating" status>

### DISCUSSION THREAD

**Discussion by qiwenzhou** [zkSync-Community-Hub/zksync-developers#525] [ZK Stack]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [X] zkEVM (era-zk_evm)
- [X] Server (zksync-era)
- [X] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [X] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [ ] Other (please specify below)

### Rust Version

1.74.0

### Operating System

- [X] Ubuntu
- [ ] MacOS
- [ ] Windows
- [ ] Other (Please specify below)

### Issue Description

when I deployed a hyperchain in my local server and used sepolia as L1, I started some transactions depositing from L1 to L2, expected that transactions will be executed on L1.But according to block explorer, those transaction status stay seal and validating.
<img width="446" alt="image" src="https://github.com/zkSync-Community-Hub/zksync-developers/assets/31309283/5d35ab49-53f4-4a87-bc08-afc0363977b2">

### Expected Behavior

transactions will be executed.

### Current Behavior

stuck in "validating"

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by bxpana** [comment]
Will check this one out and get back to you.

**Reply by qiwenzhou** [reply]
I looked deeper to it and it seemed to me that batch#2,#3... lost in DB.Still dont know why though
![YVmyMrRszr](https://github.com/zkSync-Community-Hub/zksync-developers/assets/31309283/dce08318-3a92-4b0e-b7f3-baa7a0d0c25e)
![img_v3_02ar_7cf3e5bb-e42b-4e7f-9847-21e957130chu](https://github.com/zkSync-Community-Hub/zksync-developers/assets/31309283/3b494041-9a22-401c-9c3c-3f05455cff21)

**Comment by githubdoramon** [comment]
@qiwenzhou can you please also provide some server logs?

**Reply by githubdoramon** [reply]
And, are you also running the prover, or have you created a chain without prover for tests?

**Reply by qiwenzhou** [reply]
I deployed a new chain with a prover in my local server and the same thing happend.
server log:
<img width="946" alt="image" src="https://github.com/zkSync-Community-Hub/zksync-developers/assets/31309283/849c69df-033e-4ed4-bbd8-f4a3f05d36f9">

**Reply by zk-Lumi** [reply]
Sepolia had some issues yesterday - could you try again today with proving off?

**Reply by qiwenzhou** [reply]
I still got some batch stuck in validating or executing. Is proving related to this situation, I got some proof jobs running for now, it is weird to see progress 54 has no job 
<img width="931" alt="image" src="https://github.com/zkSync-Community-Hub/zksync-developers/assets/31309283/7df6a395-2ee7-4cb3-bf01-184aedfbf664">

**Reply by qiwenzhou** [reply]
I turned off the proving and nothing changed.

**Reply by shahar4** [reply]
hey @qiwenzhou! what machine are you running the prover on? I see that it is advancing, albeit very slowly.

**Reply by qiwenzhou** [reply]
My server is AMD EPYC 7542 32-Core Processor + nvidia3090.

**Reply by bxpana** [reply]
Did you get this figured out?

**Reply by qiwenzhou** [reply]
I didn't solve this problem actually, redeployed and restart the server in the end.

**Reply by bxpana** [reply]
And that "fixed" it? You haven't had this issue since redeploying and restarting the server?
